db/migrate/20230103004613_make_experimental_hashtag_feature_default_for_new_sites.rb

```diff
@@ -0,0 +1,26 @@
+# frozen_string_literal: true
+
+class MakeExperimentalHashtagFeatureDefaultForNewSites < ActiveRecord::Migration[7.0]
+  def up
+    enable_experimental_hashtag_autocomplete =
+      DB.query_single(
+        "SELECT value FROM site_settings WHERE name = 'enable_experimental_hashtag_autocomplete'",
+      )[
+        0
+      ]
+
+    if enable_experimental_hashtag_autocomplete == "t" ||
+         enable_experimental_hashtag_autocomplete == "f"
+      return
+    end
```

Comment:
This is not necessary at all if you add an `ON CONFLICT DO NOTHING` in the next query.

Reply:
Hmm yes good point will change